flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionGraph.java

```diff
@@ -112,19 +114,24 @@
 
 	/** The log object used for debugging. */
 	static final Logger LOG = LoggerFactory.getLogger(ExecutionGraph.class);
-	
+
+	private static final int NUMBER_OF_SUCCESSFUL_CHECKPOINTS_TO_RETAIN = 1;
```

Comment:
Seems to be a rebasing error (incorrect resolve merge conflict). Will remove it.